plugins/content/shorturl/shorturl.php

```diff
@@ -81,7 +85,7 @@ public function onContentAfterSave($context, $article, $isNew)
 		
 		if (JFactory::getApplication()->isAdmin())
 		{
-			$allowedContexts = array('com_content.article');
+            $allowedContexts = array('com_content.article', 'com_k2.item');
```

Comment:
```suggestion
			$allowedContexts = array('com_content.article', 'com_k2.item');
```
Please, use the tab character instead of spaces